docs/concepts.rst

```diff
@@ -805,6 +805,14 @@ In addition to sending alerts to the addresses specified in a task's ``email`` p
 the ``sla_miss_callback`` specifies an additional ``Callable``
 object to be invoked when the SLA is not met.
 
+If you don't want to check SLAs, you can disable globally (all the DAGs) by
+setting ``check_slas=False`` under ``[core]`` section in ``airflow.cfg`` file:
+
+```ini
```

Comment:
We use restructure markup. It's markdown. Can you use `code-block` directive?

Reply:
done